### PR TITLE
Add Ping route to AI service

### PR DESCRIPTION
**Pull request by AparnaKarve**
Add a new route `/ping` which will be used by `aiops-data-collector` to check for the availability of the AI service, as part of the liveness/readiness health check for `aiops-data-collector`

(these changes would have to be cherry-picked in the other AI services as well)

### CODE REVIEW

File: tests/test_server.py [hunk 1]
```diff
@@ -66,3 +66,20 @@ def test_route_with_publisher_service_absent(mocker):
...
+    url = '/ping'
+
+    response = client.get(url)
+
+    output = {
+        "message": "Alive",
+        "status": "OK",
+        "version": VERSION
+    }
+    assert json.loads(response.get_data()) == output
```
Comment (tumido) [inline tests/test_server.py]:
```suggestion
    assert response.get_json() == output
```

**Review by tumido — CHANGES_REQUESTED** [review]
Good. This route should be present for liveliness/readiness probes for this service itself (not for data collector, see https://github.com/ManageIQ/aiops-data-collector/pull/46#pullrequestreview-261094629)

Please address the minor change in the test. The `get_data` shouldn't be used, since you're getting JSON.

File: tests/test_server.py (hunk 1 above)
Comment (AparnaKarve) [reply tests/test_server.py]: Sure. I have also made the same change in other tests.

Currently the AI Service liveness/readiness uses the root `/` GET route, where it checks for the availability of publisher -- the rationale behind that was pretty much the same (as discussed in https://github.com/ManageIQ/aiops-data-collector/pull/46#issuecomment-510973827) , basically don't let the analysis go waste, hence first ensure that we have publisher available.

Based on what we decide in https://github.com/ManageIQ/aiops-data-collector/pull/46, I can make a change here to check for publisher availability as an initial configuration check, and not include it as part of the probe.

**Comment by tumido** [comment]
Closing since repo is retired :wine_glass:

**Pull request closed without merging**